// This defines protocol for a simple server that lists files.
//
// See also the nanopb-specific options in fileproto.options.

syntax = "proto2";

message pb_msg_version {
    required uint32 version = 1;
}

message pb_msg_predict_req {
    required bytes buffer = 1;
}

message pb_msg_predict_resp {
    required uint32 found_digit = 1;
    required uint32 exec_time = 2;
    required bytes resp = 3;
}


